### ISSUE TITLE
Support a log drain that forwards to an app-level logstash instance

### ISSUE THREAD

**Issue by fancyremarker** [aptible/support#109]
Depends on #86.

**Comment by fancyremarker** [comment]
Quoting from internal issue:

> An app-level logstash would give a customer full control over log parsing and filtering without adding complexity [...]. It would also be nice to have a repo like github.com/aptible/docker-kibana for the app-level logstash that a customer could easily deploy and have all config handled by setting environment variables on that app.

**Comment by silasb** [comment]
:+1:

**Comment by wcpines** [comment]
Done
